packages/devtools_app/windows/Runner.sln

```diff
@@ -36,4 +38,6 @@ Global
 	GlobalSection(ExtensibilityGlobals) = postSolution
 		SolutionGuid = {B8A69CB0-A974-4774-9EBD-1E5EECACD186}
 	EndGlobalSection
-EndGlobal
+	GlobalSection(NestedProjects) = preSolution
+	EndGlobalSection
+EndGlobal
```

Comment:
new line at EOF?

Reply:
This file is generated by Flutter. There is a separate question of if we should be checking it in at all. Checking it in for now to avoid other people ending up with spurious diffs.
https://github.com/flutter/devtools/issues/1700